plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -542,6 +543,39 @@ private static function get_orders_by_gateway() {
 		return $orders_by_gateway_currency;
 	}
 
+	/**
+	 * Get orders origin details.
+	 * 
+	 * @return array
+	 */
+	private static function get_orders_origins() {
+		global $wpdb;
+
+		$orders_origin = $wpdb->get_results(
+			"
+			SELECT
+				meta_value as origin, COUNT( DISTINCT ( orders.id ) ) as count
+			FROM
+				wp_posts orders
+			LEFT JOIN
+				wp_postmeta order_meta ON order_meta.post_id = orders.id
+			WHERE
+				meta_key = '_created_via'
+			GROUP BY
+				meta_value;
+			"
+		);
+
+		$orders_by_origin = array();
+		foreach( $orders_origin as $origin ) {
+			$orders_by_origin[$origin->origin] = (int) $origin->count;
+		}
+
+		return array(
+			'created_via' => $orders_by_origin
+		);
```

Comment:
Minor/coding standards → should be either (trailing comma for each item in a multiline array):

```suggestion
		return array(
			'created_via' => $orders_by_origin,
		);
```

Or else a one-liner (trailing commas not required):

```suggestion
		return array( 'created_via' => $orders_by_origin );
```